src/platform/Darwin/UserDefaultUtils.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+namespace chip {
+namespace DeviceLayer {
+namespace Utils {
+
+uint16_t getUserDefaultDnssdSRPTimeout();
```

Comment:
Since this returns a time interval, it would be a good idea to add a unit of measurement to the name, like `getUserDefaultDnssdSrpTimeoutMs`

Reply:
done.